### PR TITLE
Implement #4 - fix password via CLI

### PR DESCRIPTION
**Pull request by icebob**
Add `--password=123456` CLI argument to use fix password instead of random.

@kylecarbs Currently I'm on Windows so I can't test it but I hope the code is correct. Maybe could you try it?

### CODE REVIEW

File: packages/server/src/cli.ts [hunk 1]
```diff
@@ -26,6 +26,7 @@ export class Entry extends Command {
 		version: flags.version({ char: "v" }),
 		"no-auth": flags.boolean({ default: false }),
 		"allow-http": flags.boolean({ default: false }),
+		password: flags.string(),
```
Comment (kylecarbs) [inline packages/server/src/cli.ts]: We could generate the password here and add it as a default. What do you think?

Comment (andreimc) [reply packages/server/src/cli.ts]: @kylecarbs for non interactive env's a flag should be provided I think

File: packages/server/src/cli.ts [hunk 2]
```diff
@@ -132,13 +133,17 @@ export class Entry extends Command {
...
 		});
 
-		const passwordLength = 12;
-		const possible = "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz";
-		const chars = [];
-		for (let i = 0; i < passwordLength; i++) {
-			chars.push(possible[Math.floor(Math.random() * possible.length)]);
+		let password = flags["password"];
+		if (!password) {
+			// Generate a random password
```
Comment (nhooyr) [inline packages/server/src/cli.ts]: Unrelated issue but this needs to be changed. The password should generated from a CPRNG.

File: packages/server/src/cli.ts (hunk 1 above)
Comment (icebob) [reply packages/server/src/cli.ts]: @kylecarbs Maybe, but in this case, `code-server --help` will print always a random password as a default value for `--password` argument, won't it?

File: packages/server/src/cli.ts (hunk 2 above)
Comment (icebob) [reply packages/server/src/cli.ts]: This PR is only related to adding a static password feature, not changing original password generation method.

File: packages/server/src/cli.ts (hunk 1 above)
Comment (kylecarbs) [reply packages/server/src/cli.ts]: Ah thats correct. I agree.

File: packages/server/src/cli.ts (hunk 2 above)
Comment (Multishifties) [reply packages/server/src/cli.ts]: For those reading this thread, aforementioned issue is located at https://github.com/codercom/code-server/issues/26

**Pull request merged**